sdk-base/src/main/java/com/mapbox/maps/plugin/gestures/generated/GesturesSettingsInterface.kt

```diff
@@ -97,6 +97,11 @@ interface GesturesSettingsInterface {
    * The amount by which the zoom level increases or decreases during a double-tap-to-zoom-in or double-touch-to-zoom-out gesture. 1.0 by default. Must be positive.
    */
   var zoomAnimationAmount: Float
+
+  /**
+   * Whether disable rotate gesture when pinching to zoom. false by default.
+   */
+  var disableRotateWhenPinchingToZoom: Boolean
```

Comment:
WDYT about renaming to `rotateWhenPinchingToZoomEnabled` and setting true by default ? 
IMHO 
- enabled flags are understood easier then disabled
- other flags in this settings have similar naming - e.g. `pitchEnabled` etc

Reply:
Sounds good to me. But I think `EnablePichToZoomWhenRotating` would be more readable and accurate to the current behavior.

Reply:
A bit worried that all the other options are named as `SomethingEnabled`

Reply:
iOS already added this part, will apply the same name `pinchRotateEnabled`